aiks/canvas.cc

```diff
@@ -139,6 +144,20 @@ void Canvas::ClipPath(Path path) {
   entity.SetAddsToCoverage(false);
 
   GetCurrentPass().AddEntity(std::move(entity));
+
+  ++xformation_stack_.back().stencil_depth;
+  xformation_stack_.back().contains_clips = true;
+}
+
+void Canvas::RestoreClip() {
+  Entity entity;
+  entity.SetTransformation(GetCurrentTransformation());
+  entity.SetPath({});
```

Comment:
Can you add a comment here that the the path is empty because it clear the entire render target? Just for clarity.

Reply:
Done.